app/services/onboarding.js

```diff
@@ -0,0 +1,58 @@
+import Service from '@ember/service';
+import { inject as service } from '@ember/service';
+import { TOAST_OPTIONS } from '../constants/toast-options';
+
+export default class OnboardingService extends Service {
+  @service store;
+  @service login;
+  @service toast;
+
+  constructor() {
+    super(...arguments);
+    this.login.checkAuth();
+  }
+  async signup(dataToUpdate) {
+    try {
+      let user = this.store.peekRecord('user', dataToUpdate.username);
+
+      if (!user) {
+        user = this.store.createRecord('user', {
+          username: dataToUpdate.username,
+        });
+      }
```

Comment:
Why is this?
If username is present then doesn't that mean the record is created?

Reply:
removed and check locally everything works fine